### PR TITLE
#168164932 UPDATE readme.md

### PR DESCRIPTION
**Pull request by okiti**
#### What does this PR do?
- Update the `README.md` file

#### Relevant pivotal tracker story
[#168164932](https://www.pivotaltracker.com/story/show/168164932)

### CODE REVIEW

**Comment by IAMOTZ** [comment]
Please take note of the update I did to the PR description

File: README.md [hunk 1]
```diff
@@ -1,9 +1,55 @@
...
+![GitHub closed pull requests](https://img.shields.io/github/issues-pr-closed-raw/trep-labs/realdrip-backend)
+# RealDrip Backend
+ > [Learn about The Realdrip Device](https://treplabs.co/realdrip)
+
 #### The Setup integrated the following CI/CD tools
  - TravisCI
  - Coveralls.io
- - Jest
\ No newline at end of file
+ - Jest & Suppertest
```
Comment (IAMOTZ) [inline README.md]: Jest and Supertest are not CI/CD tools, please remove them.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,55 @@
...
+ - Nodejs
+ - MongoDB
+ - Javascript
+
+## Important Dependacies
+ - [mongoose](https://mongoosejs.com/)
+ - [Jsonwebtoken](https://www.npmjs.com/package/jsonwebtoken)
+ - [validator](https://www.npmjs.com/package/validator)
+ - [nodemailer](https://nodemailer.com/)
+ - [expressJs](https://expressjs.com/)
```
Comment (IAMOTZ) [inline README.md]: Yes, these dependencies are important, however, they are no different from any other dependency in the `devDependency` section of `package.json` file e.g we have no authentication without `jwt`. Consequently, I think it's best we let the  `devDependency` section of `package.json` document the important dependency as opposed to adding them in the readme.

File: README.md [hunk 3]
```diff
@@ -1,9 +1,55 @@
...
+
+## Project setup
+```
+git clone https://github.com/TREP-LABS/realdrip-backend.git
+npm install
+```
```
Comment (IAMOTZ) [inline README.md]: If I was to give you just these two instructions, would it be enough to help you have the project up and running on your machine?
I believe you can do better on documenting the project setup, you can check [this](https://github.com/IAMOTZ/ievents#getting-started) for example.

File: README.md (hunk 2 above)
Comment (okiti) [reply README.md]: Please can you be clearer on this.

File: README.md (hunk 3 above)
Comment (okiti) [reply README.md]: okay. i'll work on that

File: README.md (hunk 1 above)
Comment (okiti) [reply README.md]: okay

**Pull request closed without merging**